src/components/UI/text/ImportantText.tsx

```diff
@@ -0,0 +1,17 @@
+import { Text, TextProps } from '@chakra-ui/react';
+import { FC } from 'react';
+
+export const ImportantText: FC<TextProps> = ({ children, ...props }) => {
+  return (
+    <Text
```

Comment:
Looks like this is a <p> tag? I wonder if this component is needed or if you could just use a header element?

Reply:
Yes, `Text` renders a `<p>` tag. I'm using it like a styled component, it's a `<p>` tag ith specific props tied to the custom theme, used across different components with color changes in some of them.